src/components/contact/SocialMedia.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFacebook, faTwitter, faInstagram } from "@fortawesome/free-brands-svg-icons";
+
+import styles from './assets/socialMedia.scss';
```

Comment:
I think the convention for css/scss file names is all lowercase

Reply:
Yeah, let's do lowercase for scss files

Reply:
Changed scss file name to lowercase